apps/src/turtle/turtle.js

```diff
@@ -212,14 +238,7 @@ Artist.prototype.init = function (config) {
   this.skin = config.skin;
   this.level = config.level;
 
-  // Preload sticker images
-  this.stickers = {};
-  for (var name in this.skin.stickers) {
-    var img = new Image();
-    img.src = this.skin.stickers[name];
-
-    this.stickers[name] = img;
-  }
+  const stickersLoaded = this.preloadAllStickerImages();
```

Comment:
Nit: not immediately clear from the method signature that this returns a promise.  Maybe remove the `stickersLoaded` const and do `preloadAllStickerImages().then(...)` below?